components/prism-core.js

```diff
@@ -1,7 +1,7 @@
+/* global WorkerGlobalScope */
```

Comment:
I get that it's convenient to just declare it here but wouldn't it be better to make an overwrite in `.eslintrc` for `components/prism-core.js` and `prism.js` to add this global variable?